### ISSUE TITLE
More report options

### ISSUE THREAD

**Issue by icare-howell** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#1479]
We have the availability to see reports based on monies; which is great. However, this system takes in so much more information that could be useful for marketing purposes. Like how they heard about us or what zip code they are from. I would like to see a way to be able to create even custom reports based on criteria the system takes in. Not just at the customer level either. Maybe I want to see how many new customers an employee added a certain day. Just would be nice to be able to generate more useful reports on multiple levels.

**Comment by Honeybee5058** [comment]
Yes this would be great! I would love an option to run reports on how many customers and also by what time people are coming
